ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/system-control-plane.j2

```diff
@@ -0,0 +1,38 @@
+{# eos - system #}
+{% if system.control_plane is arista.avd.defined %}
+!
+system control-plane
+{# control_plane tcp mss ceiling #}
+{%    if system.control_plane.tcp_mss.ipv4 is arista.avd.defined and system.control_plane.tcp_mss.ipv6 is arista.avd.defined %}
+{%       set cp_mss_cli = "tcp mss ceiling ipv4 " ~ system.control_plane.tcp_mss.ipv4 ~ " ipv6 " ~ system.control_plane.tcp_mss.ipv6 %}
+{%    elif system.control_plane.tcp_mss.ipv4 is arista.avd.defined and system.control_plane.tcp_mss.ipv6 is not arista.avd.defined %}
+{%       set cp_mss_cli = "tcp mss ceiling ipv4 " ~ system.control_plane.tcp_mss.ipv4 %}
+{%    elif system.control_plane.tcp_mss.ipv4 is not arista.avd.defined and system.control_plane.tcp_mss.ipv6 is arista.avd.defined %}
+{%       set cp_mss_cli = "tcp mss ceiling ipv6 " ~ system.control_plane.tcp_mss.ipv6 %}
+{%    endif %}
+  {{ cp_mss_cli }}
+{# control_plane access_group ipv4 #}
+{%    if system.control_plane.ipv4_access_group is arista.avd.defined %}
+{%      for acl_set in system.control_plane.ipv4_access_group | arista.avd.natural_sort %}
+{%        set cp_ipv4_access_grp = "ip access-group " ~ acl_set.acl_name %}
+{%        if acl_set.vrf is arista.avd.defined and acl_set.vrf is not none %}
```

Comment:
the `arista.avd.defined` test checks for `is defined and is not none`, so you only have to use `is arista.avd.defined`.

Reply:
Simplified to using only `is arista.avd.defined`